### DISCUSSION TITLE
Tag Accounts for Reporting Purposes, etc

### DISCUSSION THREAD

**Discussion by Lunchtime0614** [ghostfolio/ghostfolio#2386] [Q&A]
Is it possible to mark or tag an account so that you can filter the analysis (graphs etc) by certain accounts?  The use case is the ability to filter to just my retirement accounts vs standard brokerages, etc.

**Comment by dtslvr** [comment]
Hi @htims05

At the moment tagging is only possible on activity level.

**Reply by Lunchtime0614** [reply]
How do I create new tags?  Self-hosted here - I see the "EMERGENCY_FUND" tag, but no way to create new ones?

**Reply by dtslvr** [reply]
> How do I create new tags?

You need to manage it directly in the database table called _Tag_.
